examples/testplans/release/pre.json

```diff
@@ -8,11 +8,20 @@
          "description": "Navigate to:\n\n   https://github.com/avocado-framework/avocado#build-and-quality-status\n\nand make sure that all build and quality services report good indicators"},
 
 	{"name": "Avocado source is sound",
-	 "description": "On your development machine, on a freshen Avocado source to be released, run `$ make check-full`. Expected result: Make command should say OK."},
+	 "description": "On your development machine, on a freshen Avocado source to be released, run `$ make check`. Expected result: Make command should say OK."},
+
+	{"name": "Avocado time/resource sensitive tests",
+	 "description": "With a load as light as possible on your system, run `jobs/timesensitive.py`. Expected results: all tests PASSed"},
+
+	{"name": "Avocado vmimage tests",
+	 "description": "Run `jobs/vmimage.py`. Expected results: all tests PASSed"},
```

Comment:
When I'm in charge of the pre-release tests, I run this test on an AWS instance as I do not have enough bandwidth to run it at home. It would be nice to add the expected download size here, just as a reminder that it is a bandwidth consuming test.

I left it running last night, 7 tests failed, the download size for this run is ~12GB.

Reply:
Would you be able to list the failures? I only had two failures IIRC, but caused by a single image change, fixed here.

Reply:
I noticed that at a given time, the `aarch64` images were not available for the updated 10.3.4 version, so that may explain some of the failures.  Then, I will assume the `--distro=centos --distro-version=7 --arch=x86_64` failure was intermittent because I can not reproduce and the other tests for the same distro/version/arch passed.

Reply:
I ran the test again, on top of what was already downloaded, now I had 6 fails:

 (077/196) /home/fedora/avocado.master/selftests/release/vmimage.py:Provider.test_version;aarch64-aarch64-10.3.2-x86_64-5537:  FAIL: '10.3.2-20200406' not found in [] (0.91 s)
 (078/196) /home/fedora/avocado.master/selftests/release/vmimage.py:Provider.test_version;x86_64-aarch64-10.3.2-x86_64-a2c4:  FAIL: '10.3.2-20200406' not found in [] (0.63 s)

 (126/196) /home/fedora/avocado.master/selftests/release/vmimage.py:Image.test_get;aarch64-aarch64-10.3.2-x86_64-5537:  FAIL: AttributeError('Provider not available') (0.62 s)
 (127/196) /home/fedora/avocado.master/selftests/release/vmimage.py:Image.test_get;x86_64-aarch64-10.3.2-x86_64-a2c4:  FAIL: AttributeError('Provider not available') (0.82 s)

 (175/196) /home/fedora/avocado.master/selftests/release/vmimage.py:ImageFunctional.test_get;aarch64-aarch64-10.3.2-x86_64-5537:  FAIL: CmdError('avocado vmimage get --distro=debian --distro-version=10.3.2-20200406 --arch=arm64', <avocado.utils.process.CmdResult object at 0x7fe2ceaae910>) (1.33 s)
 (176/196) /home/fedora/avocado.master/selftests/release/vmimage.py:ImageFunctional.test_get;x86_64-aarch64-10.3.2-x86_64-a2c4:  FAIL: CmdError('avocado vmimage get --distro=debian --distro-version=10.3.2-20200406 --arch=amd64', <avocado.utils.process.CmdResult object at 0x7fe2cea93290>) (1.34 s)

Reply:
All failures seem to be related to the fix you sent. Let's merge it and see what happens.

Reply:
FYI: 

RESULTS    : PASS 196 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0